src/utils/streams.js

```diff
@@ -88,6 +54,8 @@ export class LengthCheckerPassThrough extends Transform {
 /**
  * This writable stream keeps all received chunks until the stream is closed.
  * Then the chunks are concatenated into a unique Buffer that can be retrieved.
+ *
+ * This is used in tests only.
```

Comment:
This is used in the storage mocks and in some streams test. I decided to keep it here for now, but might move it to test-related areas instead later...